script/setup/install-cni-windows

```diff
@@ -65,10 +65,10 @@ calculate_subnet() {
 
 # nat already exists on the Windows VM, the subnet and gateway
 # we specify should match that.
-gateway="$(powershell -c "(Get-NetIPAddress -InterfaceAlias 'vEthernet (nat)' -AddressFamily IPv4).IPAddress")"
-prefix_len="$(powershell -c "(Get-NetIPAddress -InterfaceAlias 'vEthernet (nat)' -AddressFamily IPv4).PrefixLength")"
+: ${GATEWAY:="$(powershell -c "(Get-NetIPAddress -InterfaceAlias 'vEthernet (nat)' -AddressFamily IPv4).IPAddress")"}
+: ${PREFIX_LEN:="$(powershell -c "(Get-NetIPAddress -InterfaceAlias 'vEthernet (nat)' -AddressFamily IPv4).PrefixLength")"}
 
-subnet="$(calculate_subnet "$gateway" "$prefix_len")"
+subnet="$(calculate_subnet "$GATEWAY" "$PREFIX_LEN")"
```

Comment:
Seems unrelated to arm64?
Could you consider splitting the commit?

Reply:
Not really. We used to build on Windows directly in a GH action, now we are building in Docker.